lib/benchpark/runtime.py

```diff
@@ -84,6 +84,9 @@ def __init__(self, dest):
     def bootstrap(self):
         if not self.ramble_location.exists():
             self._install_ramble()
+        else:
+            with working_dir(self.ramble_location):
+                run_command(f"git checkout {self.ramble_commit}")
```

Comment:
If our ramble is already at this commit, does the `git checkout` do nothing?

Reply:
Yes, ex:
```
(benchpark-3.12.8) [mckinsey@ruby968:ramble]$ git rev-parse HEAD
f5970aa2b16767d13650404554abaaad0cd61e1d
(benchpark-3.12.8) [mckinsey@ruby968:ramble]$ git checkout f5970aa
HEAD is now at f5970aa2 Merge pull request #816 from douglasjacobsen/hpl-template
```